recipes/pkgconfig/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pkgconfig" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "4de443a62f4b83d883a7b3895d4d4d3ca1934383ebdbeb0f2e6ab126085d812a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/matze/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose >=1.0
```

Comment:
Note: Needed for the [build]( https://github.com/matze/pkgconfig/blob/v1.1.0/setup.py#L15 ).